var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -48,22 +55,33 @@ class Plasma(MakefilePackage):
     conflicts("%xl_r")
 
     patch("remove_absolute_mkl_include.patch", when="@17.1")
+    patch("add_netlib_lapacke_detection.patch", when="@17.1")
 
     def edit(self, spec, prefix):
         # copy "make.inc.mkl-gcc" provided by default into "make.inc"
         open("make.inc", "w").write(open("make.inc.mkl-gcc").read())
 
         make_inc = FileFilter("make.inc")
 
-        if not spec.satisfies("^mkl"):
+        ld_flags = self.spec["lapack"].libs.ld_flags + " " + \
```

Comment:
`(self.spec["lapack"].libs+self.spec["blas"].libs).ld_flags` is a better way as it will deduplicate list

Reply:
fixed